<?xml version="1.0"?>
<launch>

  <arg name="processing" default="processing"/>
  <group ns="$(arg processing)">

    <!-- Cloud handler -->
    <node pkg="pmd_camboard_nano" type="pmd_camboard_nano_cloud_mesh_generator_poisson" name="cloud_mesh_generator_poisson" output="screen">
          <param name="write_to_file" type="bool" value="true"/>
          <param name="depth" type="int" value="9"/>
    </node>
  </group>
</launch>
